action.yml

```diff
@@ -32,6 +32,9 @@ inputs:
   ignore-unpublished-changes:
     description: 'Exit the workflow gracefully if package does not have a new version to publish'
     default: 'false'
+outputs:
+  published:
+    description: 'JSON formatted string with published crates in object format { name => version }'
```

Comment:
Please describe it in README.

Reply:
I will soon. Thank for review.

Reply:
Probably solved in [that commit](https://github.com/katyo/publish-crates/pull/666/commits/555baf3ec4a5aff784dcf42dc5e78ebe51c4dc47).